src/libs/elektra/kdb.c

```diff
@@ -1597,52 +1593,51 @@ static bool runGetPhase (KeySet * backends, Key * parentKey, uint16_t phase)
  * Retrieve Keys from the Key database in an atomic and universal way.
  *
  * @pre The @p handle must be a valid KDB handle as returned from kdbOpen().
- * @pre The @p returned KeySet must be a valid KeySet, e.g. constructed
- *     with ksNew().
- * @pre The @p parentKey Key must be a valid Key, e.g. constructed with
- *     keyNew().
+ * @pre The KeySet @p returned must be a valid KeySet, i.e., constructed with ksNew().
+ * @pre The KeySet @p returned must contain keys only from the `spec:/`, `dir:/`, `user:/`, `system:/`, `default:/` or `proc:/` namespaces.
+ * @pre The @p parentKey Key must be a valid Key, i.e., constructed with keyNew().
+ * @pre The @p parentKey Key must not have read-only name, value or metadata.
+ * @pre The @p parentKey Key must use the `spec:/`, `dir:/`, `user:/`, `system:/`,
+ * 	`default:/`, `proc:/` or cascading namespace.
  *
  * If you pass `NULL` or a key with read-only metadata as @p parentKey,
  * kdbGet() will fail immediately without doing anything.
  * If you pass another invalid @p parentKey, or `NULL` as @p ks or @p handle,
  * kdbGet() will set an error on @p parentKey and then return immediately.
  *
- * **Attention**: If you pass a non-NULL @p parentKey with writable metadata, kdbGet() will **always** remove
- *                any existing errors and warnings from @p parentKey.
+ * @note If you pass a non-NULL @p parentKey with writable metadata, kdbGet() will **always** remove
+ *       any existing errors and warnings from @p parentKey.
  *
- * **Important**: The KDB @p handle DOES NOT store a copy of the retrieved @p ks internally.
- *                This means if you later call kdbSet() with the same @p handle you must make sure to
- *                pass all keys from @p ks, which you do not want to remove.
+ * @warning The KDB @p handle DOES NOT store a copy of the retrieved @p ks internally.
+ *          This means if you later call kdbSet() with the same @p handle you must make sure to
+ *          pass all keys from @p ks, which you do not want to remove.
```

Comment:
What we do internally or not should not be API docu. Please only describe user-visible behavior.

```suggestion
 * @warning If you later call kdbSet() with the same @p handle you must make sure to
 *          pass all keys from @p ks, which you do not want to remove.
```

Reply:
IIRC this was taken almost 1:1 from the pre-`new-backend` docs, but sure I'll change it